amundsen_application/static/js/components/common/PopularTables/constants.ts

```diff
@@ -2,3 +2,4 @@
 export const POPULAR_TABLES_INFO_TEXT = 'These are some of the most commonly accessed tables within your organization.';
 export const POPULAR_TABLES_LABEL = 'Popular Tables';
 export const POPULAR_TABLES_SOURCE_NAME = 'popular_tables';
+export const POPULAR_TABLES_PER_PAGE = 4;
```

Comment:
Why not a property on the component itself with a default value of 4?

Reply:
This follows the pattern established for the search page and bookmarks. This might also be easier to import and test.

Reply:
I see, also just remembered that since this isn't a reusable component and only appears in one use case, its fine to be a constant and if anything would be changed from the application config.